stream_alert/rule_processor/classifier.py

```diff
@@ -217,12 +219,16 @@ def _process_log_schemas(self, payload):
             parser = parser_class(options)
 
             # Get a list of parsed records
+            LOGGER.debug('Trying schema: %s', log_name)
             parsed_data = parser.parse(schema, payload.pre_parsed_record)
 
-            LOGGER.debug('Schema: %s', schema)
+            # LOGGER.debug('Schema: %s', json.dumps(schema, indent=4))
```

Comment:
Do you want to leave this commented out? If you don't want to remove it please add a comment as to why